### ISSUE TITLE
vendor files css both included and required

### ISSUE THREAD

**Issue by pajooh** [ngbp/ngbp#208]
vendor_files:css is merged into application css file(in concat:build_css:), and also is liked (in index:build) to load.

the index:build task should modified to not link it. (remove '<%= vendor_files.css %>')

also, changing a less file just do recess:build, but concat:build_css is also required (add it to delta:less tasks)

**Comment by willw-git** [comment]
Agreed. If you do have vendor css files, as things stand they are concatenated into the main CSS file AND referenced in index.html - where they cause an error, as they haven't been copied across.

**Comment by artworkad** [comment]
+1 Yes this is nasty, I think it is related to

```
filterForCSS
```

starting in line 581 of the grunfile. Can someone open a PR for this? In build process the vendor css file shold be skipped...

**Comment by artworkad** [comment]
@willw-git @pajooh fixed this with simple file filtering that was already there actually:

```
return file.match(/^(?=.*.css)((?!vendor).)*$/);
```

please test this.

**Comment by artworkad** [comment]
@joshdmiller this issue has PR now
